### PR TITLE
Create first section of About Us Page

### LINKED ISSUE #31: Create About Us Main Section
This section splits with #22.

image

### PR DESCRIPTION
**Pull request by tylerlam1**
Resolves #31.

**Webpage version**

https://user-images.githubusercontent.com/41935461/185303500-9d138eca-8944-4590-936a-469d6d0b09e5.mp4

**Mobile Version**

https://user-images.githubusercontent.com/41935461/185303805-732dce13-1661-422a-8de8-e8ffca852d70.mp4

### CODE REVIEW

File: components/Timeline.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { TIMELINE_DATA } from "../data/timeline";
 import { UnderlineTypes } from "../utils/underlineType";
 import Heading from "./Heading";
 
-const TIMELINE_CONTAINER = "bg-white flex flex-col";
+const TIMELINE_CONTAINER = "bg-white flex flex-col lg:h-0 lg:min-height-full";
 const PURPLE_SIDE_LINES_DIR = "/svgs/timeline/left_side_lines.svg";
 
 const Timeline = () => {
```
Comment (harsohailB) [inline components/Timeline.jsx:9]: Add a little more padding to top of `Timeline` title? Also to the `Executives` title probably

Comment (harsohailB) [inline components/Timeline.jsx:9]: Before the mobile breakpoint, the images look a little weird. Maybe use `md:` breakpoint as well?

Screen Shot 2022-08-17 at 10 56 34 PM

File: pages/who-we-are/index.js [hunk 2]
```diff
@@ -26,6 +27,12 @@ const FAQDropdown = ({ prompt, content }) => {
 const WhoWeAre = () => {
   return (
     <div className="flex flex-col w-full">
```
Comment (harsohailB) [inline pages/who-we-are/index.js]:
```suggestion
    <div className="flex flex-col w-full -mt-5 md:mt-0 lg:mt-0">
```

Right now there is space in the corners

File: components/Timeline.jsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { TIMELINE_DATA } from "../data/timeline";
 import { UnderlineTypes } from "../utils/underlineType";
 import Heading from "./Heading";
 
-const TIMELINE_CONTAINER = "bg-white flex flex-col";
+const TIMELINE_CONTAINER = "bg-white flex flex-col lg:h-0 lg:min-height-full";
```
Comment (harsohailB) [inline components/Timeline.jsx]:
```suggestion
const TIMELINE_CONTAINER = "bg-white flex flex-col lg:h-0 lg:h-screen";
```

What do you think bout this?

**Review by harsohailB — CHANGES_REQUESTED** [review]
Awesome! Just minor CSS stuff

File: components/Timeline.jsx (hunk 1 above)
Comment (tylerlam1) [reply components/Timeline.jsx:9]: I made them both 2.5rem

Comment (tylerlam1) [reply components/Timeline.jsx:9]: Will do

Comment (tylerlam1) [reply components/Timeline.jsx:9]: Fixed this

**Review by harsohailB — APPROVED** [review]
We may need to go back later and implement something to lead images like medium does (blurry .... to clear) for better load times. Looks good tho!

**Pull request merged**